packages/nameguard-sdk/src/index.ts

```diff
@@ -457,7 +457,7 @@ export interface SecurePrimaryNameResult {
    * NameGuard report for the `primary_name`.
    *
    * * `null` if `primary_name_status` is `no_primary_name` (primary name is not found)
-   * * `null` if `SecurePrimaryNameOptions.computeNameGuardReport` is `false` or not provided
+   * * `null` if `SecurePrimaryNameOptions.returnNameGuardReport` is `false` or not provided
    */
   nameguard_result: NameGuardReport | null;
```

Comment:
It seems we should rename something here.

In some cases we have the param `returnNameGuardReport` that uses the phrasing "...report".
In other cases (such as here) we have the param `nameguard_result` that uses the phrasing "...result".

It seems to me that we should either use "...report" or "...result" everywhere and not use both.

My preference is to use "...report" everywhere as that's a bit more specific. What do you think?

Reply:
@lightwalker-eth @djstrong  I tried refactoring all `result`s to `report`s. There are so many places this needs to be changed that I think it requires a new PR and a careful review. I could just ctrl+f replace everything but that seems kind of wrong.

Reply:
OK!